Code/Tools/TestImpactFramework/Runtime/Common/Code/Source/Artifact/Factory/TestImpactTestEnumerationSuiteFactory.cpp

```diff
@@ -84,4 +87,74 @@ namespace TestImpact
             return testSuites;
         }
     } // namespace GTest
+
+    namespace Python
+    {
+        AZStd::vector<AZStd::pair<AZStd::string, AZStd::vector<TestEnumerationSuite>>> TestEnumerationSuitesFactory(
+            const AZStd::string& testEnumerationData)
+        {
+            const char newLineDelim = '\n';
```

Comment:
question: In this case the new line delimiter is never going to change and it's known by the majority of engineers what this is so simply using `\n` may be sufficient in my mind